python/py-cli-helpers/Portfile

```diff
@@ -14,14 +14,14 @@ long_description    CLI Helpers is a Python package that makes it easy \
                     to perform common tasks when building command-line apps. \
                     It’s a helper library for command-line interfaces.
 
-python.versions     27 34 35 36
+python.versions     27 34 35 36 37
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            cli_helpers-${version}
 
-checksums           rmd160  d9a431a99f9a605bab1837d385c5cf7d41549d0a \
-                    sha256  55353117960700dfbe000a71cda0bad1ac865e3a9999f1fa81047fa9e1322d42
+checksums           rmd160  4e1f6bb54c161a0a4eb7dc9e27eb574470e2749c \
+                    sha256  f77837c5fbcbea39e0cb782506515459a0da75465489bae35e46da7f51c5b9fc
```

Comment:
It is now customary to put in a "size" field in addition to the rmd160 and sha256.